.gitignore

```diff
@@ -9,6 +9,7 @@ node_modules
 reqs/static/js/bundle.js
 ui-dist
 ui/static/font/fontawesome-webfont*
+ui/static/font/KaTeX*
```

Comment:
Er, where are these files coming from?  I'm not seeing any logic in this code that creates these files, so I'm uncertain, but I'm probably just missing something obvious.

Reply:
Sure thing -- assuming I understand it correctly, this stems from webpack's `file-loader`. When building the styles, if webpack encounters a font referenced by the css, it'll pull it out and place it in the `static/font` directory. We don't want to commit these to the repo, though, as they're coming from a library.